src/backend/distributed/commands/citus_add_local_table_to_metadata.c

```diff
@@ -306,6 +307,17 @@ CreateCitusLocalTable(Oid relationId, bool cascadeViaForeignKeys, bool autoConve
 	ObjectAddress tableAddress = { 0 };
 	ObjectAddressSet(tableAddress, RelationRelationId, relationId);
 
+	/*
```

Comment:
As discussed offline we'll keep the sequence dependency changes as a part ot this PR. We are moving those functions because we've started to handle sequences within dependency resolution logic.